### ISSUE TITLE
@emotion + GlobalStyles: Identifier 'css' has already been declared

### ISSUE THREAD

**Issue by ben-rogerson** [ben-rogerson/twin.macro#313]
I've identified a conflict with Twins `GlobalStyles` import and the `css` import from @emotion.

If you have an import like this:  `import { Global, css } from '@emotion/react';` in the same file as the GlobalStyles import then you'll see an error like:

```shell
Module parse failed: Identifier 'css' has already been declared 
```

The workaround for now is to move the twins import `<GlobalStyles />` into another file.

Fix is coming soon.

**Comment by ben-rogerson** [comment]
Fix is in twin@2.2.3

**Comment by Knapptr** [comment]
I'm currently getting this error when using `import styled from '@emotion/styled` which I'm doing to get TS and Gatsby to work with twin.  Workaround of moving `<GlobalStyles />` works here for now.
